NBitcoin/Secp256k1/ECPubKey.cs

```diff
@@ -304,6 +305,29 @@ private bool secp256k1_eckey_pubkey_tweak_add(ECMultContext ctx, ref GE key, in
 			return true;
 		}
 
+		/// <summary>
+		/// The original function name is `secp256k1_ec_pubkey_combine`
+		/// </summary>
+		public static bool TryCombine(Context ctx, IEnumerable<ECPubKey> pubkeys, out ECPubKey? combinedPubKey)
+		{
+			if (pubkeys == null)
+				throw new ArgumentNullException(nameof(pubkeys));
+			combinedPubKey = null;
+			var count = pubkeys.Count();
+			if (count == 0)
+			{
+				return false;
+			}
+			var ptj = GEJ.Infinity;
+			foreach (var p in pubkeys)
+			{
+				ptj = ptj.Add(in p.Q);
+			}
+			var pt = ptj.ToGroupElement();
+			combinedPubKey = new ECPubKey(new GE(pt.x.Normalize(), pt.y.Normalize()), ctx);
```

Comment:
What's the difference with this code?
```c#
combinedPubKey = new ECPubKey(pt.ToGroupElement(), ctx);
```

Reply:
I wasn't aware that ECPubKey constructor was running `NormalizeVariable`, so your concern is right, running Normalize() here is unnecessary. I will change it to what you have suggested.

Reply:
@joemphilips folllow what secp256k1 is doing. If they do it, do it as well.

Reply:
it is passing generated ge to `secp256k1_pubkey_save` , I suppose that is equivalent to ECPubKey constructor. So `new ECPubKey(qj.ToGroupElement())` is equivalent.